t/t1200-stats-basic.t

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+
+test_description='Test stats collection and sending'
+
+. `dirname $0`/sharness.sh
+
+test_under_flux 2
+
+export FLUX_FRIPP_STATSD=0.0.0.0:9999
+
+test_expect_success 'create udp client program' '
+	cat <<-EOF >listen.py &&
```

Comment:
Since this script is larger than a few lines now, feel free to include it directly in `t/scripts/stats-listen.py` or something to ease future development. (Sorry, this is my fault, I should have suggested that initially)